amida-tech/indaba-client
# INBA-636 Hide custom export section

#### What does this PR do?
Removes the custom export code for MVP. 

#### Related JIRA tickets:
[INBA-636](https://jira.amida-tech.com/browse/INBA-636)

#### How should this be manually tested?
Attempt a survey export, only Quick Export should be displayed. If you see "custom export", manual testing should report a failure.

#### Background/Context

#### Screenshots (if appropriate):
